.github/buildomat/jobs/tuf-repo.sh

```diff
@@ -173,7 +173,7 @@ path = "$rot_image_a"
 [artifact.${tufaceous_board}_rot.source.archive_b]
 kind = "file"
 path = "$rot_image_b"
-[artifact.${tufaceous_board}_sp]
+[[artifact.${tufaceous_board}_sp]]
```

Comment:
It means `artifact.gimlet_sp` (etc) is now an array of items rather than a single one; it corresponds to this change:

```diff
 struct DeserializedManifest {
     system_version: SemverVersion,
     #[serde(rename = "artifact")]
-    artifacts: BTreeMap<KnownArtifactKind, DeserializedArtifactData>,
+    artifacts: BTreeMap<KnownArtifactKind, Vec<DeserializedArtifactData>>,
 }
```